MantidQt/CustomInterfaces/src/Tomography/TomographyIfaceModel.cpp

```diff
@@ -560,6 +561,19 @@ void TomographyIfaceModel::refreshLocalJobsInfo() {
   }
 }
 
+void TomographyIfaceModel::updateProcessInJobList(const qint64 pid,
+                                                  const int exitCode) {
+  // cast to string from qint64 so we can compare
+  const std::string processPID = std::to_string(static_cast<int>(pid));
```

Comment:
What happens if the PID falls outside of the range of an integer - is there any guarantees that it will always be within the range of an int?